physutils/tasks.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+import pydra
```

Comment:
This definitely adds complexity, but since physutils might be used by modules that are not based in pydra, could we try to create a safe import and make it optional, like the one we have for duecredit?

https://github.com/physiopy/phys2denoise/blob/master/phys2denoise/due.py

The idea would then to try `from pydra.mark import task`, but if it is not found, `import task` from a stub submodule of physutils. That task would be a decorator that returns the function.
(like here: https://github.com/BMRRgroup/wfTFI/blob/ec5a62cf2ffb347c752293e6234b7c89cebe711d/QSM.py#L7C1-L13C1)

Reply:
> That task would be a decorator that returns the function

What do you mean here? I'm not sure I understand how you'd like this to be done (mostly given the duecredit example).

I tried doing smth like this, but it didn't really work out
```
def mark_task_wrapper(func):
    try:
        import pydra
        logger.debug("Pydra is installed and is currently enabled.")
        return pydra.mark.task(func)
    except ImportError:
        logger.warning("Pydra is not installed and is currently disabled. Please install it to use this module.")
        return func
```

Reply:
@smoia feel free to check, I implemented it with just a wrapper and it seems to work